### PR TITLE
WIP to()  for influxdb

### PR DESCRIPTION
**Pull request by aanthony1243**

### CODE REVIEW

File: query/functions/outputs/to.go [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+	timeColIdx := execute.ColIdx(timeColLabel, columns)
+
+	if timeColIdx < 0 {
+		return errors.New("no time column detected")
+	}
+	if columns[timeColIdx].Type != flux.TTime {
+		return fmt.Errorf("column %s of type %s is not of type %s", timeColLabel, columns[timeColIdx].Type, flux.TTime)
+	}
+
+	// prepare field function if applicable and record the number of values to write per row
```
Comment (jlapacik) [inline query/functions/outputs/to.go]: Comment not applicable anymore.

File: query/functions/outputs/to.go [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+	if spec.FieldFn != nil {
+		if err = t.fn.Prepare(columns); err != nil {
+			return err
+		}
+
+	}
+
+	name := "UNKNOWN"
```
Comment (jlapacik) [inline query/functions/outputs/to.go]: Is "UNKNOWN" a special measurement to storage? If it is, should we have a constant referring to its definition in storage? Kind of like `name := storage.UnknownMeasurementName`?

Comment (aanthony1243) [reply query/functions/outputs/to.go]: no....I think what needs to happen is we error on an unknown measurement.  I'm not sure how the IDPE one works without a measurement name, but this one won't.

File: query/functions/outputs/to.go [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+	ToSignature.Params["orgID"] = semantic.String
+	ToSignature.Params["host"] = semantic.String
+	ToSignature.Params["token"] = semantic.String
+	ToSignature.Params["timeColumn"] = semantic.String
+	ToSignature.Params["tagColumns"] = semantic.Array
+	ToSignature.Params["fieldFn"] = semantic.Function
+
+	flux.RegisterFunction(ToKind, createToOpSpec, ToSignature)
+	flux.RegisterOpSpec(ToKind, func() flux.OperationSpec { return &ToOpSpec{} })
+	plan.RegisterProcedureSpec(ToKind, newToProcedure, ToKind)
```
[the remark this answers is not in the document]
Comment (aanthony1243) [reply query/functions/outputs/to.go]: I'm not finding that.  is it new?

**Pull request merged**